src/acceptance/administrator/components/com_menu/MenuItemCest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+use Step\Acceptance\Administrator\MenuItemStep;
+use Page\Acceptance\Administrator\MenuItemListPage;
+
+/**
+ * Menu Item class
+ *
+ * @category  Menu_Item
+ * @package   Administratorcomponentscom_Menu
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @link      ArticleMenuCest
+ * @since     4.0.0
+ */
+class MenuItemCest
+{
+    /**
+     * Variables Initialised
+     *
+     * menuItemTitle	string
+     * menuItemAlias	string
+     */
+    public function __construct()
+    {
+        $this->menuItemName = 'Joomla Menu Item';
+        $this->menuItemAlias = 'Joomla Menu Item Alias';
+    }
+    /**
+     * Creates a menu item with the Joomla menu manager
+     *
+     * @param \AcceptanceTester $I
+     *
+     * @throws \Exception
+     * @since  4.0.0
+     * @return void
+     */
+    public function menuItem(\AcceptanceTester $I)
+    {
+        /**
+         * Initializing values of variables
+         * using faker library
+         * string   menuItemName    name of menu item
+         * string   menuItemAlias   alias of menu item
+         */
+        $menuItemName = $this->menuItemName;
+        $menuItemAlias = $this->menuItemAlias;
+        // Call Step function to create Menu item
+        MenuItemStep::menuItem($I, $menuItemName, $menuItemAlias, '', 'Articles', 'Archived Article');
```

Comment:
if you inject the right object, you don't have to call the actions like this, rather you could simply do this
`$I->menuItem`